tfexec/errors.go

```diff
@@ -129,6 +129,11 @@ func (e *ErrVersionMismatch) Error() string {
 	return fmt.Sprintf("unexpected version %s (min: %s, max: %s)", e.Actual, e.MinInclusive, e.MaxExclusive)
 }
 
+func (e *ErrVersionMismatch) Is(target error) bool {
+	_, ok := target.(*ErrVersionMismatch)
+	return ok
+}
+
```

Comment:
I think you can just remove this code entirely, the default implementation builtin to go should handle it.

```suggestion
```

Reply:
This is actually necessary because once it goes through `Is()`, the `errors` package handles unwrapping, which otherwise isn't handled and so `errors.Is` would return false on an unwrapped error.